releases/release-1.10/release-notes-draft.md

```diff
@@ -1,3 +1,1018 @@
+
+* StorageOS volume plugin has been updated to support mount options and environments where the kubelet runs in a container and the device location should be specified. ([#58816](https://github.com/kubernetes/kubernetes/pull/58816), [@croomes]([https://github.com/croomes](https://github.com/croomes)))
+
+* Get parent dir via canonical absolute path when trying to judge mount-point, fixing a problem that caused an NFS volume with improper permissions to get stuck in `TERMINATING` status. ([#58433](https://github.com/kubernetes/kubernetes/pull/58433), [@yue9944882]([https://github.com/yue9944882](https://github.com/yue9944882)))
+
+* Clusters with GCE feature 'DiskAlphaAPI' enabled can now dynamically provision GCE PD volumes. ([#59447](https://github.com/kubernetes/kubernetes/pull/59447), [@verult]([https://github.com/verult](https://github.com/verult)))
+
+* Added `keyring` parameter for Ceph RBD provisioner. ([#58287](https://github.com/kubernetes/kubernetes/pull/58287), [@madddi]([https://github.com/madddi](https://github.com/madddi)))
+
+* Added xfsprogs to hyperkube container image. ([#56937](https://github.com/kubernetes/kubernetes/pull/56937), [@redbaron]([https://github.com/redbaron](https://github.com/redbaron)))
+
+* Improved messages user gets during and after volume resizing is done, providing a clear message to the user explaining what to do when resizing is finished. ([#58415](https://github.com/kubernetes/kubernetes/pull/58415), [@gnufied]([https://github.com/gnufied](https://github.com/gnufied)))
+
+* MountPropagation feature is now beta. As consequence, all volume mounts in containers are now "rslave" on Linux by default. To make this default work in all Linux environments you should have entire mount tree marked as shareable via "mount --make-rshared /". All Linux distributions that use systemd already have root directory mounted as rshared and hence they need not do anything. In Linux environments without systemd we  recommend running "mount --make-rshared /" during boot, before docker is started. ([#59252](https://github.com/kubernetes/kubernetes/pull/59252), [@jsafrane]([https://github.com/jsafrane](https://github.com/jsafrane)))
+
+* Volume metrics support for vSphere Cloud Provider has been added. You can now monitor available space, capacity, and used space on volumes created using vSphere. ([#59328](https://github.com/kubernetes/kubernetes/pull/59328), [@divyenpatel]([https://github.com/divyenpatel](https://github.com/divyenpatel)))
+
+* Emit number of bound and unbound persistent volumes as Metrics. This PR adds four kinds of Volume Metrics for kube-controller-manager: bound PVC numbers, unbound PVC numbers, bound PV numbers and unbound PV numbers. The PVC metrics use namespace as dimension and the PV metrics use Sto
```

Comment:
Can we link to the docs here? The linked PR is the implementation PR and contains implementation details (which users don't need to care about).

Docs link, which will work when 1.10 is released: https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#subresources

Reply:
Err, I'm not sure why Github is showing this comment for line 541. I meant this for the subresources release note: https://github.com/kubernetes/sig-release/pull/92/files#diff-fbb7988712c575e744f5f9d12b9f4392R617